content/en/security_platform/_index.md

```diff
@@ -6,34 +6,13 @@ aliases:
-
-{{< img src="security_monitoring/explorer.png" alt="Security Signals Explorer"  >}}
-
-Enable runtime security and continuous Compliance Monitoring for your containers and hosts. Use **File Integrity Monitoring (FIM)** to watch for changes to key files and directories on hosts or containers in real-time, or **Continuous Compliance** to check against your containers and Kubernetes clusters to find configuration issues, as defined in the popular CIS compliance benchmarks for Docker and Kubernetes.
-
-{{< img src="compliance_monitoring/compliance-dashboard.png" alt="Datadog Compliance Monitoring" >}}
+The Datadog Security Platform unifies developer, operation, and security teams through one platform. Use a single dashboard to display DevOps content, business metrics, and security content. Detect threats, like a targeted attack, an IP communicating with your systems which matches a threat intel list, or an insecure configuration, to your application and infrastructure in real time, and notify your team of security issues by email, slack, Jira, PagerDuty, or a webhook.
 
 {{< whatsnext desc="This section includes the following topics:">}}
-  {{< nextlink href="/security_monitoring/getting_started">}}<u>Getting Started</u>: Learn the main concepts of Datadog Security Monitoring, how to enable threat detection, and discover out of the box threat detection rules.{{< /nextlink >}}
-  {{< nextlink href="/security_monitoring/detection_rules">}}<u>Threat Detection Rules</u>: Establish Detection Rules to define conditional logic for detecting security threats and suspicious behavior within all ingested logs.{{< /nextlink >}}
-  {{< nextlink href="/security_monitoring/default_rules">}}<u>Default Rules</u>: Out of the box Threat Detection Rules to flag attacker techniques and potential misconfigurations to immediately improve your security posture.{{< /nextlink >}}
-  {{< nextlink href="/security_monitoring/explorer">}}<u>Signals Explorer</u>: Search, visualize, and triage your Security Signals.{{< /nextlink >}}
-
+  {{< nextlink href="/security_platform/security_monitoring">}}<u>Security Monitoring</u>: Detect threats, like a targeted attack, an IP communicating with your systems which matches a threat intel list, or an insecure configuration, to your application and infrastructure in real time.{{< /nextlink >}}
+  {{< nextlink href="/security_platform/cspm">}}<u>Cloud Security Posture Management</u>: Track the compliance posture of your production environment, automate audit evidence collection, and catch misconfigurations that leave your organization vulnerable to attacks.{{< /nextlink >}}
```

Comment:
Edit: Track the security hygiene and compliance posture of your production environment, automate audit evidence collection, and catch misconfigurations that leave your organization vulnerable to attacks.